go/acronym/acronym.go

```diff
@@ -4,31 +4,27 @@ package acronym
 
 import "strings"
 
-// Generates and acronym our of a string
-// parameters
+// Abbreviate generates an acronym representing a string parameter.
 // * inputString : base string for the acronym. Will be split into words
-// 				   using ' ' or '-' as a separator. Each word will then
-//				   be used to compute the returned acronym
+//				   using ' ' or '-' as a separator. Each word will then
```

Comment:
Here you are mixing tabs and spaces, which makes the diffs difficult to read. Please try to configure `goimports` or `gofmt` to run at least before commit (or maybe also on each save, which is my case).

Reply:
Thanks @ivajloip , however, goimports is already running on each save. It seems to ignore the comments (just checked on an other file, by putting spaces and tabs mixed both in codes and comments). I am using vim-go, would you suggest another plugin (if there is any) ?